src/components/bookmark-button/index.tsx

```diff
@@ -24,7 +26,12 @@ export default function BookmarkButton({ isBookmarked }: BookmarkButtonProps) {
 			onClick={() => {
 				// TODO use the create / destroy methods in the client
 				// or render dialog to prompt auth if not auth'd
-				console.log('Bookmark clicked!')
+				toast({
+					title: `Bookmark Added`,
+					description: 'This tutorial has been added to your bookmarks.', // TODO update to use the tutorial name here
```

Comment:
Will adjust this while connecting things with real data. Presumably we will have the tutorial name / id available in this function in order to update the API.